comment/api/serializers.py

```diff
@@ -23,10 +26,13 @@ def get_profile_model():
 def get_user_fields():
     user_model = get_user_model()
     fields = user_model._meta.get_fields()
+    api_fields = list(settings.COMMENT_USER_API_FIELDS) + ['profile']
+    # remove duplicates, preserve order
+    api_fields = list(OrderedDict.fromkeys(api_fields))
```

Comment:
Please use `set` for removing duplication.
- More readability.
- No extra import statement needed.
- No comment needed to explain what this line is doing.

```suggestion
    api_fields = list(set(api_fields))
```

Reply:
`sets` don't necessarily prevent orders and that is what the comment says. we would want to preserve the order in which custom fields have been specified.

Reply:
I am not sure that I understood what you mean, but why would we care about the orders? We are getting unique values from a list, right?

Reply:
i have set the fields to `[`a`, 'b']` and now my frontend code is written in a way that tends to care about the order in which these fields are returned.

Reply:
Hmmm! You might be confused here? The returned value is json not an ordered list, so these fields will be used in json object like so:
```python
fields = ['a', 'b']

response = {
    'a': a_value,
    'b': b_value
}
```

That means keeping the same order has no meaning.